docs/development/angular/performance-cookbook.md

```diff
@@ -0,0 +1,274 @@
+# Performance boosting and optimization of Angular applications cookbook
+
+## Table of contents
+
+* [Theoretical introduction](#theoretical-introduction)
+  * [Event loop](#event-loop)
+  * [zone.js](#zone.js)
+  * [Change detection](#change-detection)
+    * [OnPush change detection strategy](#onpush-change-detection-strategy)
+* [Recipes](#recipes)
+  * [Architecture](development/angular/performance-cookbook/architecture.md)
+  * [Change detection](development/angular/performance-cookbook/change-detection.md)
+  * [zone.js](development/angular/performance-cookbook/zonejs.md)
+  * [Pipes](development/angular/performance-cookbook/pipes.md)
+  * [Routing](development/angular/performance-cookbook/routing.md)
+  * [Lazy loading](development/angular/performance-cookbook/lazy-loading.md)
+  * [Managing events](development/angular/performance-cookbook/managing-events.md)
+  * [Constant bindings and structural directives](development/angular/performance-cookbook/constant-bindings-and-structural-directives.md)
+  * [RxJS](development/angular/performance-cookbook/rxjs.md)
+  * [V8](development/angular/performance-cookbook/v8.md)
+  * [Multithreading](development/angular/performance-cookbook/multithreading.md)
+  * [Memory management](development/angular/performance-cookbook/memory-management.md)
+  * [Dynamic components](development/angular/performance-cookbook/dynamic-components.md)
+  * [Angular 8+ (Ivy)](development/angular/performance-cookbook/angular8-ivy.md)
+  * [SSR](development/angular/performance-cookbook/ssr.md)
+  * [PWA](development/angular/performance-cookbook/pwa.md)
```

Comment:
I believe that TOC is excessive.
1. We have one in the sidebar and the 2nd one is duplicate
2. Docsify automatically generates submenu for header 2nd level (h2).
3. It is hard to maintain TOC in file